uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
@niiquaye we should probably use this file as like a template for other Cmakelists in the repo as it has all the necessary things to build, run tests, etc. if we keep all of the cmake stuff looking somewhat the same, itll be eaasier to maintain

Reply:
i updated the template in our README. all cmakelists should follow this imo